code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -384,6 +384,7 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("white jumpsuit", /obj/item/clothing/under/color/white, 3), \
 	new/datum/stack_recipe("white shoes", /obj/item/clothing/shoes/sneakers/white, 2), \
 	new/datum/stack_recipe("white scarf", /obj/item/clothing/neck/scarf, 1), \
+	new/datum/stack_recipe("loincloth", /obj/item/clothing/under/costume/loincloth/cloth, 2), /*SKYRAT EDIT ADDITION*/ \
```

Comment:
but these ones still look like diapers from the front view
![chrome_pm7T7rB3Jn](https://user-images.githubusercontent.com/4081722/159590112-2f78a82b-5034-4020-b2e2-4f8d268e4c1c.png)

Reply:
is there any way we can try to break up the sprite a bit to make it look less like that?